app/src/main/java/edu/uc/mille5bm/smartnutrition/MainActivity.kt

```diff
@@ -1,3 +1,8 @@
+/** 
+*   Contains the information gathered from the NutritionInfoActivity, CalorieCounterActivity, and HealthTrackerActivity classes 
+*
+*
+*/
```

Comment:
KDoc typically goes above a class or a function, but not above a package.   One exception is copyright, but this KDoc isn't copyright.